### ISSUE TITLE
NPM doesn't add JSDom dependency request to package-lock.json

### ISSUE THREAD

**Issue by cafca** [FixMyBerlin/fixmy.frontend#673]
JSDOM lists `requests^2.88.2` as a dependency but running `npm install` doesn't add it to `package-lock.json`. I have added `request` as a top-level dev-dependency for now, but that's not a sustainable solution.

Steps to reproduce:

1. Uninstall my added top-level `requests` dependency with `npm uninstall requests`
2. Run `npm install`
3. Search for `"requests"` in `package-lock.json`

Expected result:

`requests` is listed in JSDOM's `dependencies` field.

Actual result:

JSDOM's `dependencies` field does not contain requests.

**Comment by s-pic** [comment]
@cafca Mind me taking a look at this?

**Comment by ohrie** [comment]
I don't understand the problem correctly. Where do we need `jsdom`? It does not appear in the `package.json`. Only Jest needs it. But adds it correctly. The test run fine.

**Comment by s-pic** [comment]
Guess this regards to the lock file, not the package file

**Comment by ohrie** [comment]
Yeah, you're right. But when `jsdom` doesn't include `request`, why should be add it? ^^ Where does jsdom or out project need it, didn't find it.
